web/content/docs/benchmarks/heatconduction/heatconduction-line_source_term.pandoc

```diff
@@ -0,0 +1,118 @@
++++
+author = "Thomas Fischer"
+date = "2019-10-01T11:54:02+01:00"
+title = "Heatconduction (Line Source Term)"
+weight = 121
+project = "Parabolic/T/1D_line_source_term_tests/line_source_term.prj"
+
+[menu]
+  [menu.benchmarks]
+    parent = "heatconduction"
+
++++
+
+{{< data-link >}}
+
+## Equations
+
+We consider the Poisson equation:
+$$
+\begin{equation}
+\nabla\cdot(\nabla T) + Q_T = 0 \quad \text{in }\Omega
+\end{equation}$$
+w.r.t Dirichlet-type boundary conditions
+$$
+\eqalign{
+T(x) = 0 &\quad \text{on }\Gamma_D,\cr
+}
+$$
+where $T$ could be temperature, the subscripts $D$ denotes the Dirichlet-type
+boundary conditions. Here, the temperature distribution under the impact of a
+line shaped source term should be studied.
+
+## Problem Specifications and Analytical Solution
+
+In OGS there are several benchmarks for line source terms in 2d and 3d domains
+available. Here, some of the 3d benchmarks are described.
+
+### Cylindrical domain
+
+The Poisson equation on cylindrical domain of heigth $1$ and radius
+$r=1$ is solved. In the following figure the geometry, partly semi-transparent,
+is sketched. Furthermore, the mesh resolution is shown in the cylindrical domain
+within the first quadrant of the coordinate system. In the second quadrant the
+simulated temperature distribution is depicted.
+
+{{< img src="../LineSourceTermFigures/temperature_distribution_line_source_term_in_cylinder.png" >}}
```

Comment:
Here just add a line in the 3D figure, showing where the temperatures were compared. I guess it does not matter. Just to be clear.

Reply:
Added the line to the figure where the temperature profile is plotted.